docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -25,9 +25,9 @@ A software-defined asset includes the following:
 
 - An <PyObject object="AssetKey" />, which is a handle for referring to the asset.
 - A set of upstream asset keys, which refer to assets that the contents of the software-defined asset are derived from.
-- An [op](/concepts/ops-jobs-graphs/ops), which is a function responsible for computing the contents of the asset from its upstream dependencies.
+- A Python function which is responsible for computing the contents of the asset from its upstream dependencies.
```

Comment:
This is maybe a more controversial change, so if you don't think it makes sense that's totally cool. putting myself in the shoes of a brand new user, I envision my path through the docs to be 1) tutorial, 2) concept pages. Specifically, I would start with the assets concept page. and being presented with the concept of ops right away would be pretty intimidating (thoughts like "ok so i need to go read that concept page first?", "maybe i won't understand assets if i dont understand ops" etc)